### DISCUSSION TITLE
Default Payment Request and Invoice to send email

### DISCUSSION THREAD

**Discussion by manakill** [btcpayserver/btcpayserver#2161] [Ideas / Feature Requests]
I use other invoicing platforms and it is most common for an email to be sent to the customer when an Invoice or Payment Request is created. Simple email with a button to go to the payment page. Could be a global toggle per Store. Or even a toggle per Invoice/Payment Request.

**Comment by pavlenex** [comment]
I think this is a reasonable request and it's been requested quite often to be built in. Hoping more community members may voice their ideas here so that we can prioritize it if there's interest. You can always do this via [Transmuter](https://docs.btcpayserver.org/Transmuter/EmailReceiptsPreset/) if you want to thinker, but functionality like this should be out of the box in the UI imo.

**Reply by manakill** [reply]
> 
> 
> I think this is a reasonable request and it's been requested quite often to be built in. Hoping more community members may voice their ideas here so that we can prioritize it if there's interest. You can always do this via [Transmuter](https://docs.btcpayserver.org/Transmuter/EmailReceiptsPreset/) if you want to thinker, but functionality like this should be out of the box in the UI imo.

Ah yes, Transmuter would do part of the job, but not fully.
I see on GitCoin people can create Jobs for coders to do and pay with bitcoin. Do you think that would be a good option to use to focus some coder talent on getting it done? I feel basics such as sending emails like this, is much needed, in BTCPayServer

**Reply by pavlenex** [reply]
I'm not familiar with what that is, but if someone wants to build this functionality in, I see no problem with that though note pull requests outside our roadmap may take longer to review as they aren't priority.

**Comment by manakill** [comment]
@pavlenex I think I've found a solution that will work for us. We are using Invoice Ninja now to send Invoices to clients, and that takes care of sending the email with the pretty pdf invoice. 

Now BTCPayServer is being added as a Payment Gateway! This should result in a streamlined, sleek customer experience, that can be replicated fairly easily, for other micro businesses like myself.
https://github.com/invoiceninja/invoiceninja/issues/5386 

There should still be better options built into BTCPayServer probably, for people just running BTCPayServer with no invoicing software.

**Reply by pavlenex** [reply]
That's some really awesome news @manakill we have this feature in mind, and I agree it's quite reasonable to have something like this added in the future. Thanks for the info 🚀
